docs/jetpack-compose.md

```diff
@@ -1,7 +1,37 @@
 # Jetpack Compose
 
-Although Jetpack Compose has not hit 1.0 yet, ensuring that Mavericks will be compatible with it is extremely important. Alongside Compose there is value in using Mavericks as a container for business logic that will pair well with a tree of composable UI nodes.
+Mavericks is still useful in a Jetpack Compose world. With Jetpack Compose, you can use Compose for all of your UI but Mavericks for your business logic, data fetching, interface to your dependency injection object graph, etc.
 
-We haven't settled on a final API yet but have already begin to explore what one might look like. [Here](https://gist.github.com/gpeal/5bed618a843d4d71bb1518fa8317c5df) are some snippets of working code that enables you to use Mavericks with Compose today. It is likely that the final API will look extremely similar to this.
+To use Mavericks with Jetpack Compose, add the `com.airbnb.android:mavericks-compose` artifact (TBD on versioning).
 
-When Compose and the associated tooling (Android Studio, Android Gradle Plugin, etc) are closer to being finalized, we will release an official Compose artifact.
+To get started, create a MavericksViewModel just like you always have. Now, with the `mavericks-compose` artifact, you can get or create a ViewModel in a composable like this:
+
+```kotlin
+@Composable
+fun MyComponent() {
+    val viewModel: CounterViewModel = mavericksViewModel()
+    val count by viewModel.collectState(CounterState::count)
+    Button(onClick = activityScopedViewModel::incrementCount) {
```

Comment:
what is `activityScopedViewModel` referencing here.? should it be `viewModel`?

Reply:
Copy and paste :)